### PR TITLE
test: add test to useParagonTheme hook and paragon utils

### PR DESCRIPTION
**Pull request by dcoa**
**Description:**

This is a testing contribution to Theming Configuration in Runtime, covering useParagonTheme hook, and the utils file.

Context: https://github.com/openedx/frontend-platform/pull/440

C.C @adamstankiewicz

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @dcoa!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @dcoa - is this ready for review?

cc: @adamstankiewicz

**Comment by dcoa** [comment]
Hi @mphilbrick211, yes, it is

**Comment by mphilbrick211** [comment]
Hi @openedx/fed-bom! Would someone be able to review / merge this for us?

File: src/react/hooks/paragon/utils.test.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+describe('removeExistingLinks', () => {
+  afterEach(() => {
+    document.head.innerHTML = '';
+    jest.clearAllMocks();
+  });
+
+  it('should remove all the links in the DOM', () => {
+    document.head.innerHTML = `
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@edx/paragon@$21.0.0/dist/core.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@edx/brand@$2.0.0Version/dist/core.min.css">
```
[the remark this answers is not in the document]
Comment (dcoa) [reply src/react/hooks/paragon/utils.test.js]: Yes, I fixed it, thanks

**Comment by dcoa** [comment]
Hi, @adamstankiewicz, any additional feedback?

**Comment by mphilbrick211** [comment]
> Hi, @adamstankiewicz, any additional feedback?

Hi @adamstankiewicz! Friendly ping on this.

**Comment by mphilbrick211** [comment]
Hi @openedx/fed-bom @adamstankiewicz! Is this something you could please review / merge for us? Thanks!

**Review by adamstankiewicz — APPROVED** [review]
Thanks for the contribution, @dcoa! Apologies for the delayed review.

**Comment by openedx-webhooks** [comment]
@dcoa 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-platform&entry.867055334=https://github.com/openedx/frontend-platform/pull/525&entry.1484655318=https://github.com/dcoa&entry.752974735=2023-07-25+09:58&entry.1917517419=2023-12-09+19:21&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**